urbansearch/clustering/preprocessing.py

```diff
@@ -0,0 +1,40 @@
+#Tokenizes, Stems, removes stop words and counts
+
+import re
+
+from nltk.tokenize import word_tokenize
+from nltk.stem.snowball import SnowballStemmer
+from nltk.corpus import stopwords
+from collections import Counter
+
+class PreProcessor:
+    def __init__(self, lang='dutch'):
+        self.lang = lang
+        self.stemmer = SnowballStemmer(self.lang)
+
+    def preProcess(self, text):
+        text = self.stripPunctuations(text) #expects string
+        text = self.tokenize(text) #transfers to array
+        text = self.stem(text) #Do this before stripping ('Het' is not removed, 'het' is)
+        text = self.stripWords(text) #expects array
+        text = self.count(text)
+        return text
+
+    def stem(self, text):
```

Comment:
This is a perfect example of a method that can be reduced to list comprehension, like the method below.

Something like: `return [self.stemmer.stem(w) for w in text]`